src/content/docs/en/guides/imports.mdx

```diff
@@ -200,6 +200,11 @@ Additionally, glob patterns must begin with one of the following:
  
 
 [Read more about the glob pattern syntax](https://github.com/mrmlnc/fast-glob#pattern-syntax).
+
+#### `Astro.glob()` vs Content Collections
+
+[Content collections](/en/guides/content-collections/) were introduced in `astro@2.0.0`. It's the best way to work with Markdown and MDX files in an Astro project. If your directory contains only Markdown and MDX files, it's best to use Content collections and the `src/content` directory.
```

Comment:
✋ Can I veto “content-based files”?

I know we sometimes say “content-based websites”, but here I think we can either say “content files” or just “content”.

```suggestion
[Content collections](/en/guides/content-collections/) provide a [`getCollection()` API](/en/reference/api-reference/#getcollection) for loading multiple files instead of `Astro.glob()`. If your content files (e.g. Markdown, MDX, Markdoc) are located in collections within the `src/content/` directory, use `getCollection()` to [query a collection](/en/guides/content-collections/#querying-collections) and return content entries.
```

Reply:
I don't know if this is better or worse but here I goes
```suggestion
[Content collections](/en/guides/content-collections/) is a new content layer API to fetch local content files (e.g. Markdown, MDX, Markdoc) introduced in `astro@2.0.0`. You can use `getCollection()` instead of `Astro.glob` to [query a collection](/en/guides/content-collections/#querying-collections) and return content entries.

Reply:
Love it when Chris pulls out his veto! 

I'm going to go with that!

Reply:
Kevin, I went with Chris' suggestion! In particular, I think we're standardizing on not mentioning how things are new when they are just "things that exist" now. So, I would have taken that part entirely out. We typically use the `added in` tag for that, so if they really care, the info is available in a way that makes them read fewer words.  :sweat_smile: